_layouts/people.html

```diff
@@ -274,138 +244,130 @@ <h2>Details</h2>
         {% if ols-1-people contains username %}
             {% assign roles = '' %}
             {% if ols-1-participants contains username %}
-                {% capture roles %}{{ roles }}, project lead{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 project lead{% endcapture %}
             {% endif %}
             {% if ols-1-mentors contains username %}
-                {% capture roles %}{{ roles }}, mentor{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 mentor{% endcapture %}
             {% endif %}
             {% if ols-1-experts contains username %}
-                {% capture roles %}{{ roles }}, expert{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 expert{% endcapture %}
             {% endif %}
             {% if ols-1-speakers contains username %}
-                {% capture roles %}{{ roles }}, speaker{% endcapture %}
-            {% endif %}
-            {% if ols-1-hosts contains username %}
-                {% capture roles %}{{ roles }}, call host{% endcapture %}
+                {% capture roles %}{{ roles }},OLS-1 speaker{% endcapture %}
             {% endif %}
             {% if ols-1-organizers contains username %}
-                {% capture roles %}{{ roles }}, organizer{% endcapture %}
+                {% capture roles %}{{ roles }}, OLS-1 organizer{% endcapture %}
             {% endif %}
             {% capture ols-roles %}{{ ols-roles }} / OLS-1 {{ roles | remove_first: ', ' }}{% endcapture %}
```

Comment:
Be sure that the cohort appears correctly there

Reply:
I do not understand why OLS-1 is not written 2 times in the `ols-roles` variable. Any guess?

Reply:
Hi @bebatut. If we output `ols-roles`, the cohort (OLS-1) shows twice.
Like this...

**OLS-1 OLS-1 organizer.**

But if we output `roles`, the result is...
**OLS-1 organizer.**

Reply:
But we should display `ols-roles` not `roles` in the people cards. `roles` will be the role in the last cohort they participated, not in all cohorts

Reply:
That's correct. I had actually tried to do it this way at first...but `ols-roles` sort of looked messy for people who had many roles in the different cohorts.

If you don't mind how it looks, then all we have to do is remove the cohort I specified on each line.

![Screenshot (348)](https://user-images.githubusercontent.com/105166953/198600067-bce776fa-e4ad-439b-9d23-3ede426ac79e.png)

![Screenshot (349)](https://user-images.githubusercontent.com/105166953/198601838-198ada14-51c8-4bd5-9cb3-2d054f020965.png)

Reply:
Removing the redundancy in the cohort name would help already :smile: 

> That's correct. I had actually tried to do it this way at first...but ols-roles sort of looked messy for people who had many roles in the different cohorts.

It is good to show that people come back :smile:

Reply:
😂😂😂 That's smart.

Reply:
I'll make the fix right away...